configs/config/pretrain/byol/byol_8node_resnet.yaml

```diff
@@ -66,10 +67,8 @@ config:
         ZERO_INIT_RESIDUAL: True
     HEAD:
       PARAMS: [
-        ["mlp", {"dims": [2048, 4096], "use_relu": True, "use_bn": True}],
-        ["mlp", {"dims": [4096, 256]}],
-        ["mlp", {"dims": [256, 4096], "use_relu": True, "use_bn": True}],
-        ["mlp", {"dims": [4096, 256]}],
+        ["mlp", {"dims": [2048, 4096, 256], "use_relu": True, "use_bn": True}],
```

Comment:
This was the bug I found.

When specifying an MLP, the last layer will always be a linear unless you set `skip_last_layer_relu_bn=False`.

Hence our model was 

```
Linear(2048 -> 4096)
Linear(4096 -> 256)
Linear(256 -> 4096)
Linear(4096 -> 256)
```

instead of 
 ```
Linear(2048 -> 4096)
Batchnorm
Relu
Linear(4096 -> 256)
Linear(256 -> 4096)
Batchnorm
Relu
Linear(4096 -> 256)
```